src/goGetPackage.ts

```diff
@@ -23,24 +22,17 @@ export function goGetPackage() {
 		return;
 	}
 
-	const goRuntimePath = getBinPath('go');
-	if (!goRuntimePath) {
-		return vscode.window.showErrorMessage(`Failed to run "go get" to get package as the "go" binary cannot be found in either GOROOT(${process.env['GOROOT']}) or PATH(${envPath})`);
-	}
-
 	const env = Object.assign({}, process.env, { GOPATH: getCurrentGoPath() });
+	const result = await goGet(importPath, ['-v'], { env });
 
-	cp.execFile(goRuntimePath, ['get', '-v', importPath], { env }, (err, stdout, stderr) => {
-		// go get -v uses stderr to write output regardless of success or failure
-		if (stderr !== '') {
-			outputChannel.show();
-			outputChannel.clear();
-			outputChannel.appendLine(stderr);
-			buildCode();
-			return;
-		}
-
-		// go get -v doesn't write anything when the package already exists
+	// go get -v doesn't write anything when the package already exists
+	if (result.stderr === '') {
```

Comment:
What is the reason to flip the `if` condition here?
If package already exists, there is no need to build the code or show the output channel

Reply:
Fixed to add a return. I flipped it because I always find it easier to understand code when the uncommon case is in the if conditions.